eslint.config.mjs

```diff
@@ -1,12 +1,20 @@
-// @ts-check
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import tseslint from "typescript-eslint";
+import googleLint from "eslint-config-google";
 
-import eslint from '@eslint/js';
-import tseslint from 'typescript-eslint';
 
-export default tseslint.config(
-  eslint.configs.recommended,
+export default [
+  {files: ["**/*.{js,mjs,cjs,ts}"]},
+  {languageOptions: { globals: {...globals.browser, ...globals.node} }},
+  pluginJs.configs.recommended,
   ...tseslint.configs.recommended,
+  {
+    plugins: {
+      googleLint
```

Comment:
Remove google lint since it doesn't seem to actually work - it is also is deprecated and we shouldn't be using deprecated packages/linters